generics/helpers/slackCommunications.js

```diff
@@ -162,7 +162,107 @@ const rubricErrorLogs = function (errorMessage) {
   }
 }
 
+const badRequestForNonLoggedInUsers = function (errorMessage) {
```

Comment:
Change it to badSharedLinkAccessAttemptAlert. Also, add configuration in env to turn these alerts on or off. @ajithtunerlabs

Reply:
Also please share a sample slack error message that will come to me.